src/main/java/Todo.java

```diff
@@ -0,0 +1,11 @@
+public class Todo extends Task {
```

Comment:
Perhaps put the class in a package?
See [Coding Standard - Package and Import Statements](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements).